narwhals/functions.py

```diff
@@ -1179,12 +1178,7 @@ def sum(*columns: str) -> Expr:
         |    0  3  4.8     |
         └──────────────────┘
     """
-    return Expr(
-        lambda plx: plx.col(*columns).sum(),
-        is_order_dependent=False,
-        changes_length=False,
-        aggregates=True,
-    )
+    return col(*columns).sum()
```

Comment:
🤩

Reply:
😄 not sure why i didn't write it like this in the first place 😳